embedchain/llm/gpt4all.py

```diff
@@ -19,13 +19,28 @@ def get_llm_model_answer(self, prompt):
     @staticmethod
     def _get_instance(model):
         try:
-            from gpt4all import GPT4All
+            from langchain.llms import GPT4All
         except ModuleNotFoundError:
             raise ModuleNotFoundError(
-                "The GPT4All python package is not installed. Please install it with `pip install --upgrade embedchain[opensource]`" # noqa E501
+                "The GPT4All python package is not installed. Please install it with `pip install --upgrade embedchain[opensource]`"  # noqa E501
             ) from None
 
-        return GPT4All(model_name=model)
+        return GPT4All(
+            temp=BaseLlmConfig().temperature,
+            allow_download=True,
+            model=model,
+            n_threads=8,
+            max_tokens=BaseLlmConfig().max_tokens,
+            n_parts=8,
+            backend="llama",
+            seed=0,
+            f16_kv=False,
+            n_batch=8,
+            embedding=False,
+            logits_all=False,
+            vocab_only=False,
+            use_mlock=False,
```

Comment:
Do we really need to hard code these params? Can we skip these and let the GPT4All to pick the default ones?

Reply:
@deshraj So sorry for the inactivity. will check if we can remove these hardcoded values.